flipperdevices/flipperzero-firmware
# [FL-2515] Keep backlight on for updater ops

# What's new

- [FL-2515] Updater: Keep backlight on for updater ops

# Verification 

- Build update package, install
- Check that BL doesn't turn off while updater is running

# Checklist (For Reviewer)



[FL-2515]: https://flipperzero.atlassian.net/browse/FL-2515?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
